kimchi/src/prover.rs

```diff
@@ -1044,6 +1044,35 @@ where
         //~ 1. Squeeze the Fq-sponge and absorb the result with the Fr-Sponge.
         fr_sponge.absorb(&fq_sponge.digest());
 
+        //~ 1. Compute evaluations for the previous recursion challenges.
```

Comment:
I don't think this phrasing is great, but I guess it's not going to be great until we specify the recursion anyway